js/w3c/style.js

```diff
@@ -1,55 +1,115 @@
 // CONFIGURATION
 //  - specStatus: the short code for the specification's maturity level or type (required)
-
+"use strict";
 define(
     ["core/utils"],
-    function (utils) {
-        return {
-            run:    function (conf, doc, cb, msg) {
-                msg.pub("start", "w3c/style");
-                if (!conf.specStatus) msg.pub("error", "Configuration 'specStatus' is not set, required for w3c/style");
-                var statStyle = conf.specStatus;
-                if (statStyle === "FPWD"    ||
-                    statStyle === "LC"      ||
-                    statStyle === "WD-NOTE" ||
-                    statStyle === "LC-NOTE") statStyle = "WD";
-                if (statStyle === "FPWD-NOTE") statStyle = "WG-NOTE";
-                if (statStyle === "finding" || statStyle === "draft-finding") statStyle = "base";
-                var css = "https://";
-                if (statStyle === "unofficial") {
-                    css += "www.w3.org/StyleSheets/TR/w3c-unofficial";
-                }
-                else if (statStyle === "base") {
-                    css += "www.w3.org/StyleSheets/TR/base";
-                }
-                else if (statStyle === "CG-DRAFT" || statStyle === "CG-FINAL" ||
-                         statStyle === "BG-DRAFT" || statStyle === "BG-FINAL") {
-                    // note: normally, the ".css" is not used in W3C, but here specifically it clashes
-                    // with a PNG of the same base name. CONNEG must die.
-                    css += "www.w3.org/community/src/css/spec/" + statStyle.toLowerCase() + ".css";
-                }
-                else if (statStyle === "webspec") {
-                    css = "https://specs.webplatform.org/assets/css/kraken.css";
-                    $('<link rel="icon" href="https://specs.webplatform.org/assets/img/icon.png">', doc)
-                        .appendTo($("head"));
-                    $(doc.createElement("script"))
-                        .attr({ async: "async", src: "https://specs.webplatform.org/assets/js/kraken.js"})
-                        .appendTo($("head"))
-                        ;
-                }
-                else if (conf.useExperimentalStyles) {
-                    css += "www.w3.org/StyleSheets/TR/2016/W3C-" + statStyle;
-                }
-                else {
-                    css += "www.w3.org/StyleSheets/TR/W3C-" + statStyle;
-                }
-                utils.linkCSS(doc, css);
-                msg.pub("end", "w3c/style");
-                cb();
-            }
+    function(utils) {
+      function attachFixupScript(doc, version){
+        var script = doc.createElement("script");
+        script.async = "async";
+        var helperScript = "https://www.w3.org/scripts/TR/{version}/fixup.js"
+          .replace("{version}", version);
+        script.src = helperScript;
+        doc.head.appendChild(script);
```

Comment:
Can we please change this to doc. body.appendChild so that pubrules will pass?  Then LGTM.